Mathlib/Data/Real/EReal.lean

```diff
@@ -29,6 +29,9 @@ but it is sometimes convenient to have.
 An ad hoc multiplication is defined, for which `EReal` is a `CommMonoidWithZero`. We make the
 choice that `0 * x = x * 0 = 0` for any `x` (while the other cases are defined non-ambiguously).
 This does not distribute with addition, as `⊥ = ⊥ + ⊤ = 1*⊥ + (-1)*⊥ ≠ (1 - 1) * ⊥ = 0 * ⊥ = 0`.
+Distributivity `x * (y + z) = x * y + x * z` is recovered in the case where either `0 ≤ x < ⊤`,
+see `left_distrib_of_nonneg_of_ne_top`, or `0 ≤ y, z`, see `left_distrib_of_nonneg` (similarily
```

Comment:
```suggestion
see `Ereal.left_distrib_of_nonneg_of_ne_top`, or `0 ≤ y, z`, see
`Ereal.left_distrib_of_nonneg` (similarily
```
can you check more generally that in the doc all declarations are called with their full name (including the namespace)? This makes the website links work.